docs/source/tls.rst

```diff
@@ -113,6 +115,42 @@ To find the fingerprint of a given certificate, use the following OpenSSL comman
 
 SHA256 and SHA512 fingerprints are supported.  SHA1 fingerprints are not supported due to the insecurity of the SHA1 algorithm.
 
+Skip Certificate Validation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Depending on the specifics of your environment(s), if you need to turn off certificate validation, you can add a `SkipReceptorNamesCheck` key-value pair in your configuration file for `tls-server`, `tls-config`, or both.
+
+.. code-block:: yaml
+
+    ---
+    - node:
+        id: bar
+
+    - log-level:
+        level: Debug
+
+    - tls-client:
```

Comment:
I think for clarity we can just have a truncated config snippet here, with just the tls-client and tls-server portions. We do this elsewhere in the docs.

Reply:
should i keep the `node.id` line as well? just to highlight the importance between the `node.id` and the certificates now?